.circleci/config.yml.j2

```
@@ -129,7 +129,7 @@ jobs:
           command: |
             set -o pipefail
             helm template --kube-version=v{{ version }} -f values.yaml --set global.baseDomain=example.com . |
-            kubent --cluster=false --helm2=false --helm3=false --filename -
+            kubent --cluster=false --helm=false --filename -
```

Looks like we need these options:
```
$ kubent --help 2>&1 | grep helm
      --helm2   enable Helm v2 collector (default true)
      --helm3   enable Helm v3 collector (default true)
```
```suggestion
            kubent --cluster=false --helm2=false --helm3=false --filename -
```